### DISCUSSION TITLE
user defined labels for collapsed actions and feedbacks

### DISCUSSION THREAD

**Discussion by SmutjeHannes** [bitfocus/companion#2977] [Ideas]
I love the feature to collapse the actions and feedbacks to smaller chunks but it would be amazing to set the text to something the user could define. A descriptive name of the action/feedback instead of just the kind of action/feedback.

**Comment by Julusian** [comment]
This has been possible since 3.3.0.
Click the pencil icon to the right of the action name to switch it to the editable field
![image](https://github.com/user-attachments/assets/2e437a1e-688d-4228-98ea-12cabc184187)

**Comment by SmutjeHannes** [comment]
Perfect! Thank you! How did I not see it?!
